objects/src/transaction/mod.rs

```diff
@@ -23,3 +24,6 @@ pub use prepared_tx::PreparedTransaction;
 pub use proven_tx::ProvenTransaction;
 pub use tx_result::{FinalAccountStub, TransactionResult};
 pub use tx_witness::TransactionWitness;
+
+#[cfg(any(test, feature = "testing"))]
+pub mod utils;
```

Comment:
If these are used only for testing, should we name them `test-utils` or something like that?

Reply:
It's not only used for testing, we use it locally in the crate as well however we need to expose this publicly for testing in other crates.  I've updated the flag to `#[cfg(feature = "testing")]` to better represent the requirements.